tests/datasource/conftest.py

```diff
@@ -1,5 +1,6 @@
 import os
 
+import pandas as pd
```

Comment:
@Shinnnyshinshin Is this Pandas import needed?  Thanks.

Reply:
Nope not at all. Thank you for catching @alexsherstinsky